actors/builtin/miner/expiration_queue.go

```diff
@@ -283,15 +284,15 @@ func (q ExpirationQueue) RescheduleAllAsFaults(faultExpiration abi.ChainEpoch) e
 		return nil
 	}
 
-	// Add rescheduled sectors to new expiration as early-terminating and faulty.
-	allRescheduled, err := bitfield.MultiMerge(rescheduledSectors...)
+	// Add missing sectors to new expiration as early-terminating and faulty.
+	allNotFound, err := bitfield.MultiMerge(rescheduledSectors...)
```

Comment:
Sorry, I think the github diff collapsing caused me to request a rename there that's not very good. It was fine before.